.github/workflows/bors.yml

```diff
@@ -205,9 +206,10 @@ jobs:
             bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
 
       - name: upload cache
-        # We only upload the cache if the build ran (either succeeding or failing),
-        # but not if it was skipped.
-        if: ${{ always() && steps.build.outcome == 'success' || steps.build.outcome == 'failure' }}
+        # We only upload the cache if the build started (whether succeeding, failing, or cancelled)
+        # but not if any earlier step failed.
```

Comment:
```suggestion
        # but not if any earlier step failed or was cancelled.
```

Reply:
@adomani, is that what you meant?

Reply:
This was not what I had in mind, but it certainly is good to mention it!

I think that I was confused and in my mind I was still thinking that you were splitting on the `build` step, not on the `get` step!